.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,38 @@
 
+---
+**PR checklist**
+
+The following checklist shall help the PR's author, the reviewers and maintainers to ensure the quality of this project.
+It bases on our contributors guidelines, especially the ["writing code" section](https://github.com/junit-pioneer/junit-pioneer/blob/master/CONTRIBUTING.md#writing-code).
+It shall help to check for completion of the listed points.
+If a point does not apply to the PR changes, the entry can just be marked as done. 
+
+Documentation (general)
+ 
+* [ ] There is documentation (Javadoc and site documentation; added or updated)
+* [ ] There are implementation information to describe _why_ non-obvious source code / solution got implemented
```

Comment:
Correction: There are implementation information -> There is implementation information

Reply:
Hm yeah. At the start of the sentence I wrote plural, at the end singular. Changed completely to singular